network/src/test/kotlin/com/tapptitude/network/SessionInterceptorTest.kt

```diff
@@ -0,0 +1,88 @@
+package com.tapptitude.network
+
+import com.tapptitude.network.fake.FakeSessionManager
+import com.tapptitude.network.fake.IgnoredCallback
+import com.tapptitude.network.interceptor.SessionInterceptor
+import com.tapptitude.session.SessionManager
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.After
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+
+/**
+ * Underneath, Retrofit2 uses OkHttp for requests.
+ *
+ * Thus, if we want to test an interceptor, it is enough to use a OkHttp object.
+ * No need to create an instance of retrofit
+ */
+internal class SessionInterceptorTest {
+    private lateinit var interceptor: SessionInterceptor
```

Comment:
I prefer leaving an empty line between class declaration and first field/method. Makes it feel a bit cleaner :D